initial-commit-tf.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+TF_REPO_NAME=dataworks-repo-template-terraform
+NEW_REPO_NAME=$1
+
+git config --global user.name "${GIT_USERNAME}"
+git config --global user.email "${GIT_EMAIL}"
+
+git clone https://github.com/dwp/$NEW_REPO_NAME
+cd $NEW_REPO_NAME
+
+if [ ! -z 'grep -rnw $TF_REPO_NAME ci' ]; then
```

Comment:
That's not what this does. it looks for a folder existing called `ci`, and if so renames files inside that folder.  It always exists as it's in the template repo...

Reply:
If it always exists, then why do you need a conditional check for it? Also, this would also trigger if there was just a file named `ci`, right? I guess it doesn't matter, though, as it'd just do nothing. But we don't check for it being a folder called `ci` until line 15 as far as I can see. This conditional seems to be pointless, if the `ci` folder is always expected to exist, and it doesn't actually prove what it sets out to prove.

Reply:
When you put it like that, I agree. I've removed it as it is not required.